src/LaunchDarkly/Server/Evaluate.hs

```diff
@@ -1,17 +1,17 @@
 module LaunchDarkly.Server.Evaluate where
 
-import           Control.Lens                        ((%~))
+import           Data.Generics.Internal.VL           (over)
```

Comment:
`Data.Generics.Internal.VL` is not exposed until `generic-lens` `2.0.0.0`, and the current minimum in `package.yaml` is `1.1.0.0`.